.travis.yml

```
@@ -5,13 +5,16 @@ node_js:
 
 services:
   - docker
-
+env:
+  global:
+    - GROUP=deploy.razee.io
+    - VERSION=v1alpha2
 script:
   # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
   - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low; else npx audit-ci --low || true; fi
   - npm run lint
   - npm test
-  - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}" .
+  - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}" --build-arg GROUP=${GROUP_ARG} --build-arg VERSION=${VERSION_ARG} .
```

```suggestion
  - docker build --rm -t "quay.io/razee/remoteresources3:${TRAVIS_COMMIT}" --build-arg GROUP_ARG=${GROUP} --build-arg VERSION_ARG=${VERSION} .
```
i believe you meant to do this?